src/devicelock/devicelock.cpp

```diff
@@ -199,7 +199,9 @@ bool DeviceLock::runPlugin(const QStringList &args)
     qWarning() << process.readAllStandardError();
 #endif
 
-    return process.exitCode() == 0;
+    if (process.exitCode() == 0) return OK;
```

Comment:
..although, since there are positive error codes that also mean failure now, you might be right.